contracts/src/rollup/MockRollup.sol

```diff
@@ -50,21 +47,33 @@ contract MockRollup {
         validators[msg.sender] = true;
     }
 
+    ///@notice deregisters msg.sender as validator
+    function deRegisterValidator() external {
+        require(validators[msg.sender], "MockRollup.registerValidator: Validator already registered");
+        require(!blacklist[msg.sender], "MockRollup.registerValidator: Validator blacklisted");
+        validators[msg.sender] = false;
+    }
+
     /**
      * @notice a function for validators to post a commitment to a blob on behalf of the rollup
      * @param blobHeader the blob header
      * @param blobVerificationProof the blob verification proof
      */
     function postCommitment(
         IEigenDAServiceManager.BlobHeader memory blobHeader, 
-        EigenDABlobUtils.BlobVerificationProof memory blobVerificationProof
+        EigenDABlobUtils.BlobVerificationProof memory blobVerificationProof,
+        bytes32 blobParamsHashInput
```

Comment:
What is the point of taking this as parameter if it's rederived from blobHeader below?